Description: Adaption of upstream FreeBSD source.
 GNU/kFreeBSD resorts to a compatibility library.
 .
 The call mkstemp(3) needs a longer template in Glibc.
Author: Mats Erik Andersson <debian@gisladisker.se>
Forwarded: not-needed
Last-Update: 2011-01-20

--- freebsd-8.1.orig/usr.sbin/edquota/edquota.c	2010-06-14 04:09:06.000000000 +0200
+++ freebsd-8.1/usr.sbin/edquota/edquota.c	2011-01-20 19:28:30.000000000 +0100
@@ -65,6 +65,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <bsd/string.h>
 #include <unistd.h>
 #include "pathnames.h"
 
diff -Naur freebsd-8.1.orig/usr.sbin/edquota/Makefile /tmp/freebsd-8.1/usr.sbin/edquota/Makefile
--- freebsd-8.1.orig/usr.sbin/edquota/Makefile	2010-06-14 04:09:06.000000000 +0200
+++ freebsd-8.1/usr.sbin/edquota/Makefile	2011-01-20 19:28:30.000000000 +0100
@@ -4,6 +4,8 @@
 PROG=	edquota
 MAN=	edquota.8
 
+LDADD=	-lbsd
+
 WARNS?=	4
 
 .include <bsd.prog.mk>
--- freebsd-8.1.orig/usr.sbin/edquota/pathnames.h	2010-06-14 04:09:06.000000000 +0200
+++ freebsd-8.1/usr.sbin/edquota/pathnames.h	2011-01-20 19:28:30.000000000 +0100
@@ -33,4 +33,4 @@
 #include <paths.h>
 
 #undef _PATH_TMP
-#define	_PATH_TMP	"/tmp/EdP.aXXXXX"
+#define	_PATH_TMP	"/tmp/EdP.aXXXXXX"
